extensions/dacpac/src/test/testContext.ts

```diff
@@ -285,6 +285,7 @@ export function createViewContext(): ViewTestContext {
 		modelBuilder: {
 			listView: undefined!,
 			radioCardGroup: undefined!,
+			doughnutChart: undefined!,
```

Comment:
Also - please use `donut` instead of `doughnut` - donut is the more commonly used spelling for this type of chart.